### PR TITLE
Move Lightweight Trained Neural Network version to 2.4

### PR DESCRIPTION
**Pull request by mrodozov**
Update lwtnn version to 2.4

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: lwtnn.spec [hunk 1]
```diff
@@ -1,17 +1,32 @@
-### RPM external lwtnn 1.0
+### RPM external lwtnn 2.4
+
 Source: https://github.com/lwtnn/lwtnn/archive/v%{realversion}.tar.gz
-BuildRequires: py2-pkgconfig
+BuildRequires: py2-pkgconfig ninja cmake
 Requires: eigen boost
-Patch0: lwtnn-1.0-boost-fix
```
Comment (davidlt) [inline lwtnn.spec]: The patch is not removed in CMSDIST. Please remove it.

File: lwtnn.spec [hunk 2]
```diff
@@ -1,17 +1,32 @@
-### RPM external lwtnn 1.0
+### RPM external lwtnn 2.4
+
 Source: https://github.com/lwtnn/lwtnn/archive/v%{realversion}.tar.gz
-BuildRequires: py2-pkgconfig
+BuildRequires: py2-pkgconfig ninja cmake
```
Comment (davidlt) [inline lwtnn.spec]: Remove `py2-pkgconfig` as it's not used anymore. It was used for old makefiles: `makefile:CXXFLAGS     += $(shell pkg-config eigen3 --cflags)`

**Review by davidlt — CHANGES_REQUESTED** [review]

**Comment by mrodozov** [comment]
please abort

**Comment by mrodozov** [comment]
please test

**Pull request merged**